pkg/scheduler/schedule_one.go

```diff
@@ -1128,6 +1130,9 @@ func updatePod(ctx context.Context, client clientset.Interface, pod *v1.Pod, con
 	}
 	if nnnNeedsUpdate {
 		podStatusCopy.NominatedNodeName = nominatingInfo.NominatedNodeName
+		if utilfeature.DefaultFeatureGate.Enabled(features.PodObservedGenerationTracking) {
+			podStatusCopy.ObservedGeneration = pod.Generation
```

Comment:
updating the nominated node name is the only place I could find in the scheduler where it makes an update to the pod status outside of pod conditions, but it's not obvious to me if outputting observedGeneration here is actually helpful?

Reply:
Yeah, I think we can skip it. That let's us keep the definition of ObservedGeneration in the status tied to the Kubelet.

Reply:
Thanks! Dropped this commit